src/cache/cache.rs

```diff
@@ -155,10 +155,10 @@ impl CacheRead {
         pool: &tokio::runtime::Handle,
     ) -> Result<()>
     where
-        T: IntoIterator<Item = (String, PathBuf)> + Send + Sync + 'static,
+        T: IntoIterator<Item = (String, PathBuf, bool)> + Send + Sync + 'static,
```

Comment:
I am not a big fan of tuples, especially when they accumulate primitives which are non-descriptive

Reply:
Ok, that makes sense 👍